src/gui/folderwizard.cpp

```diff
@@ -483,7 +483,8 @@ FolderWizardSelectiveSync::FolderWizardSelectiveSync(const AccountPtr &account)
     layout->addWidget(_selectiveSync);
 
     if (Theme::instance()->showVirtualFilesOption() && bestAvailableVfsMode() != Vfs::Off) {
-        _virtualFilesCheckBox = new QCheckBox(tr("Use virtual files instead of downloading content immediately (experimental)"));
+        _virtualFilesCheckBox = new QCheckBox(tr("Use virtual files instead of downloading content immediately %1").arg(
+                                                  Utility::isWindows() ? tr("(tech preview)") : tr("(experimental)")));
```

Comment:
Check if `bestAvailableVfsMode()` is the `WindowsCfApi` one. Otherwise older windows versions will show tech preview here even though they're using suffix files.